### DISCUSSION TITLE
Cefsharp 114 JavascriptObjectRepository.ResolveObject WPF property are not visible in JS

### DISCUSSION THREAD

**Discussion by kelemenattila** [cefsharp/CefSharp#4554] [Q&A]
Hi,
I have a solution in .Net 6 with cefsharp 114. I'm using JSObject to communicate between JS and WPF.  I have an old solution with .Net Framework 4.5.2 and there works with an older Cef.
I tried to reproduce the issue in the MinimalExample and unfortunately same. Method works, property not. Tried with simple property and full property (get and set for both)

![Screenshot_25](https://github.com/cefsharp/CefSharp/assets/48238058/0e8eb5c2-318d-4288-92e2-11226abfca05)

I tried with Register and ResolveObject event to.... Same behavior. 

this.InitializeComponent();
      this.Browser.JavascriptObjectRepository.Settings.LegacyBindingEnabled = true;
      this.Browser.JavascriptObjectRepository.ResolveObject += (s, e) =>
      {
        e.ObjectRepository.Register("shellMeta", new ShellMetaClass() { FullProp = "2.0.0.0", SimpleProp = "2.3.1.0" });
      };
      
      this.Browser.JavascriptObjectRepository.Register("shellMeta1", new ShellMetaClass() { FullProp = "2.0.0.0", SimpleProp="2.3.1.0" }, BindingOptions.DefaultBinder);

I hope can be resolve.
Thank you

**Comment by amaitland** [comment]
Not supported in `.NET 6`. See https://github.com/cefsharp/CefSharp/discussions/4273#discussioncomment-3884369

You will need to stick with `.Net 4.x` if you need this feature. For new app development using the `async` binding is recommended.

**Reply by kelemenattila** [reply]
Ok. Thank you for your answer :) . I will use async binding than
